DogTales/src/player.cpp

```diff
@@ -5,7 +5,7 @@ Player::Player(glm::vec2 const world_space) : m_world_space(world_space) { m_spr
 void Player::tick(bave::Seconds const dt) {
 
 	m_physics.tick(dt);
-	m_sprite.transform.position = m_physics.m_position;
+	m_sprite.transform.position = m_physics.position;
 
 	handle_wall_collision();
```

Comment:
Since `handle_wall_collision` clamps the player sprite's position to the bottom of the screen, after that point `m_physics.position` and `m_sprite.transform.position` keep diverging indefinitely, effectively pinning the player to the ground no matter what (jumping from -1000 to -990 will still get clamped to -360 or whatever).

For time being I'd suggest setting `m_physics.position = m_sprite.transform.position` after the clamping, to synchronize them again.

Reply:
fixed!